ruby/README.md

```diff
@@ -645,21 +645,25 @@ result = hash.map { |k, v| v + 1 }
 result = hash.map { |_, v| v + 1 }
 ```
 
-### Use '{}' when passing a hash as an argument, but not for keyword args.
+### Do not use a hash literal to destructure keyword args.
 ```ruby
 def foo(a:, b:)
-#...
+# ...
 end
 
-foo({a: 1, b: 2}) # bad
-foo(a: 1, b: 2) # good
+foo({ a: 1, b: 2 }) # bad
+foo(**{ a: 1, b: 2 }) # better
```

Comment:
It's unlikely someone would have a literal here. Let's show that example in its common form:
```
options = { a: 1, b: 2 }
foo(**options)
```
Hmm, is the above really better than
```
foo(options)
```
?

Reply:
I agree with the change to make the example more clear.

The main point here is that `foo(options)` will break with Ruby 3, but `foo(**options)` won't.

Reply:
That makes sense. That's exactly what `**` was invented for: take this hash and spread it over the keyword args.